corehq/apps/app_manager/models.py

```diff
@@ -2713,12 +2713,27 @@ def validate_for_build(self):
         return errors
 
 
+class ReportGraphConfig(DocumentSchema):
```

Comment:
and I'm not all that familiar with GraphConfiguration & related classes, so that comment is more "please explain this"  than "you should do things differently"